modules/scanning-images-by-using-secured-clusters.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * operating/examine-images-for-vulnerabilities.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="scanning-images-by-using-secured-clusters_{context}"]
+= Scanning images by using secured clusters
+
+To scan images by using the secured clusters instead of the Central services, you can use the delegated image scanning feature.
+
+A new delegated scanning configuration specifies the registries from which you can delegate image scans. For images that Sensor observes, you can use the delegated registry configuration to delegate scans from no registries, all registries, or specific registries.
+
+To enable delegation of scans by using the `roxctl` CLI, Jenkins plugin, or API, you must also specify a destination cluster and source registry.
+
+.Prerequisites
+
+* You have installed Scanner in the secured cluster to scan images.
++
+[NOTE]
+====
+Enabling Scanner-slim is supported on {ocp} and Kubernetes secured clusters.
+====
+
+.Procedure
+
+. In the {product-title-short} portal, click *Platform Configuration -> Clusters*.
+. In the *Clusters* view header, click *Delegated scanning*.
+. In the *Delegated Image Scanning* page, provide the following information:
+
+** *Delegate scanning for*: To choose the scope of the image delegation, select one of the following options:
+*** *None*: The default option. This option specifies that the secured clusters do _not_ scan any images, except for images from the {ocp} integrated registry.
```

Comment:
nit: recognize this was just copied, but based on [the comment here](https://github.com/openshift/openshift-docs/pull/80974#discussion_r1747177617), the preferred way to refer to the integrated registry seems to be:

```suggestion
*** *None*: The default option. This option specifies that the secured clusters do _not_ scan any images, except for images from the integrated OpenShift image registry.
```